Tests/WorkspaceTests/InitTests.swift

```diff
@@ -317,6 +316,30 @@ class InitTests: XCTestCase {
         }
     }
 
+    func testDefaultPlatform() throws {
+        #if !os(macOS)
+        // Platform default versions are only set on macOS
+        try XCTSkipIf(true, "test is only supported on macOS")
```

Comment:
is this really needed?

Reply:
Right now it would still be empty if you ran on linux, so i think until that part changes we'd have to skip? is there another way of writing this test that you're suggesting could avoid this?